### ISSUE TITLE
[Accessibility]: va-accordion buttons have insufficient color contrast

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/vets-design-system-documentation#5929]
### Point of contact

@bacitracin

### How severe do you think this issue is?

Medium

### Severity rationale

Contrast could be better, but it's still usable.

### URL where issue occurs

https://design.va.gov/storybook/?path=/docs/uswds-va-accordion--docs

### Component or pattern (if applicable)

va-accordion

### Environment details

OS: Mac
Browser: Chrome

### Steps to reproduce

Came up in accessibility testing for staging review of BMT 2's Foreign Medical Program feature. This shows up on the [va letters page](https://staging.va.gov/records/download-va-letters/letters) and is reproducible on Storybook

The light grey of the accordion item buttons against white (background) has a contrast ratio of 1.1:1.

<img width="1388" height="834" alt="Image" src="https://github.com/user-attachments/assets/7c04d4b5-9b8b-4b6c-84f1-2be090e5a836" />

### Type of issue

- [ ] Axe-core
- [ ] Cognitive | user flows & navigating
- [x] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Content style guide
- [ ] Focus
- [ ] Headings
- [ ] Keyboard
- [ ] Link and button labels
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Screenreader | assistive tech & device support
- [ ] Semantic HTML
- [ ] URL standards
- [ ] Zoom or Magnification
- [ ] Something else
